packages/schema/src/config/common.ts

```diff
@@ -139,6 +139,15 @@ export default defineUntypedSchema({
     ]
   },
 
+  /**
+   * The directory where Nuxt will store the generated files when running `nuxt analyze`.
+   *
+   * If a relative path is specified, it will be relative to your `rootDir`.
+   */
+  analyzeDir: {
+    $resolve: async (val, get) => resolve(await get('rootDir'), val || '.nuxt-analyze')
```

Comment:
This dir is not gitignored by default in any starter template. Cannot we use buildDir `.nuxt/analyze`?

Reply:
I was going `.nuxt/analyze`, but changed it to this as `.nuxt` will be cleaned out on every start, while analyzes should better be persisted so we could compare with different builds.

Reply:
We could add it to gitignore after is get shipped.

Reply:
This is how DevTools likes like atm:
<img width="922" alt="image" src="https://user-images.githubusercontent.com/11247099/233615774-966df25d-e52c-4ea2-b673-da90c5ff8da2.png">

Reply:
`.nuxt` (buildDir) should finally not be cleaned up entirely. Because this way we are also removing `.nitro/cache` and other files that can be particularly useful between builds to be kept. I think would be worth fixing this and not introducing another top-level directory.

Reply:
We can also use `node_modules/.nuxt/analzye` as another workaround solution in meanwhile. Would that work?

Reply:
I will explore on it :)